src/index.html

```diff
@@ -15,8 +15,109 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <!-- LOGO -->
+      <a
+        href="logo"
+        class="home__link"
+      >
+        <img
+          src="images\logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute for the `img` tag should describe the image content. 'logo' is too general. Try to describe what logo it is. Refer to the checklist: '<code>alt</code> attribute should describe the image content'.